gravitee-apim-console-webui/src/shared/utils/simple-validator.ts

```diff
@@ -22,6 +22,13 @@ export const isUnique = (paths: string[]): ValidatorFn | null => {
   };
 };
 
+export const isUniqueAndDoesNotMatchDefaultValue = (values: string[], defaultValue: string): ValidatorFn | null => {
+  return (control: AbstractControl): ValidationErrors | null => {
+    const formControlValue = control.value;
+    return formControlValue !== defaultValue && values.includes(formControlValue) ? { isUnique: true } : null;
```

Comment:
```suggestion
    return formControlValue !== defaultValue && values.includes(formControlValue) ? { isUniqueAndIsNotDefaultValue: true } : null;
```

Reply:
my method naming is probably not better, but I wonder if the returned object should reflect the name of the method, imho it would be easier to understand

Reply:
Yeah naming is one of the hardest parts of coding :). I'll use your suggestion as it's clearer I think

Reply:
Could you do the suggested change?